lib/core/realtime/notifier.js

```diff
@@ -372,72 +390,80 @@ class NotifierController {
 
       const action = connectionId ? 'notify' : 'broadcast';
 
-      this.kuzzle.entryPoints.dispatch(action, {
+      this.kuzzle.entryPoint.dispatch(action, {
         channels,
         connectionId,
         payload: updated
       });
     }
-    catch(error) {
+    catch (error) {
       this.kuzzle.log.error(error);
     }
   }
 
   /**
    * Broadcasts a notification about a document change or a
-   * real-time message
+   * real-time message.
+   *
+   * When this method is called from the node who received
+   * the request triggering notification, then every channel is notified
+   * regardless the "cluster" option.
+   *
+   * When this method is called from the cluster synchronization,
+   * then only the channel having the "cluster: true" option are notified
```

Comment:
```suggestion
   * then only the channel having the "replicate: true" option are notified
```

Reply:
The option is named `cluster` for channel because it has more meaning to have "cluster" channels and not "replicate" channels